features/bevragen/persoon/verblijfplaats/protocollering-vraag.feature

```diff
@@ -0,0 +1,67 @@
+#language: nl
+
+Functionaliteit: protocollering van de vraag voor verblijfplaats
+
```

Comment:
Moet in dit bestand ook nog een Rule en bijbehorend scenario worden gedefinieerd voor het vastleggen van geleverde velden? Die is immers niet gelijk aan die van de gevraagde velden.

Reply:
ik heb -op verzoek van @CathyDingemanse - de feature gesplitst in een feature voor de vraag en een feature voor het antwoord (geleverd). De geleverd (antwoord) features staan per type adres, omdat hierbij de inhoud van de persoonsgegevens (Gegeven stappen) bepalend zijn.